From 6ca53b6d6519d52021e642230bb51ae7834b3e67 Mon Sep 17 00:00:00 2001
From: Cristian Stoica <cristian.stoica@nxp.com>
Date: Tue, 9 Feb 2016 12:11:32 +0200
Subject: [PATCH 45/48] cryptodev: change signature for conversion functions

These functions are called with const BIGNUMs, so we change the
signatures to avoid compilation warnings

Signed-off-by: Cristian Stoica <cristian.stoica@nxp.com>
---
 crypto/engine/eng_cryptodev.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/crypto/engine/eng_cryptodev.c b/crypto/engine/eng_cryptodev.c
index b9c7ff3..58e539c 100644
--- a/crypto/engine/eng_cryptodev.c
+++ b/crypto/engine/eng_cryptodev.c
@@ -146,7 +146,7 @@ const EVP_CIPHER cryptodev_tls12_aes_256_cbc_hmac_sha1;
 const EVP_CIPHER cryptodev_tls12_aes_128_cbc_hmac_sha256;
 const EVP_CIPHER cryptodev_tls12_aes_256_cbc_hmac_sha256;
 
-inline int spcf_bn2bin(BIGNUM *bn, unsigned char **bin, int *bin_len)
+inline int spcf_bn2bin(const BIGNUM *bn, unsigned char **bin, int *bin_len)
 {
     int len;
     unsigned char *p;
@@ -168,7 +168,7 @@ inline int spcf_bn2bin(BIGNUM *bn, unsigned char **bin, int *bin_len)
     return 0;
 }
 
-inline int spcf_bn2bin_ex(BIGNUM *bn, unsigned char **bin, int *bin_len)
+inline int spcf_bn2bin_ex(const BIGNUM *bn, unsigned char **bin, int *bin_len)
 {
     int len;
     unsigned char *p;
-- 
2.7.3

